internal/cephfs/volume.go

```diff
@@ -134,6 +140,44 @@ func createVolume(ctx context.Context, volOptions *volumeOptions, cr *util.Crede
 	return nil
 }
 
+func resizeVolume(ctx context.Context, volOptions *volumeOptions, cr *util.Credentials, volID volumeID, bytesQuota int64) error {
+	supported := false
+	ok := false
+
+	if supported, ok = resizeSupportedList[volOptions.ClusterID]; supported || !ok {
+		args := []string{
+			"fs",
+			"subvolume",
+			"resize",
+			volOptions.FsName,
+			string(volID),
+			strconv.FormatInt(bytesQuota, 10),
+			"--group_name",
+			volOptions.SubvolumeGroup,
+			"-m", volOptions.Monitors,
+			"-c", util.CephConfigPath,
+			"-n", cephEntityClientPrefix + cr.ID,
+			"--keyfile=" + cr.KeyFile,
+		}
+
+		err := execCommandErr(
+			ctx,
+			"ceph",
+			args[:]...)
+
+		if err == nil {
+			resizeSupportedList[volOptions.ClusterID] = true
+			return nil
+		}
+		if !strings.Contains(err.Error(), inValidCommmand) {
```

Comment:
Some comments here would help. In case the command is invalid, fall-through and do the resize with `createVolume()` instead.